mava/wrappers/jumanji.py

```diff
@@ -197,40 +203,67 @@ def modify_timestep(self, timestep: TimeStep) -> TimeStep[Observation]:
             action_mask=timestep.observation.action_mask,
             step_count=jnp.repeat(timestep.observation.step_count, self.num_agents),
         )
-        if self._use_individual_rewards:
-            # The environment returns a list of individual rewards and these are used as is.
-            return timestep.replace(observation=modified_observation)
+        # Whether or not aggregate the list of individual rewards.
+        reward = aggregate_rewards(timestep.reward, self.num_agents, self._use_individual_rewards)
```

Comment:
I must admit I am not a massive fan of the `not` here. But I prefer it over having the conditional in the aggregation function. What do you think of just having the config option be `aggregate_rewards` instead of `use_individual_rewards`? Then we could change the conditional here to `if self._aggregate_rewards`.
```suggestion
        if not self._use_individual_rewards:
            reward = aggregate_rewards(timestep.reward, self.num_agents)
```

Reply:
eh either way I prefer this :smile:

Reply:
I chose the second option to use `aggregate_rewards`  instead of the not which is less confusing 😅